instrumentation/opentelemetry_ecto/lib/opentelemetry_ecto.ex

```diff
@@ -201,6 +205,14 @@ defmodule OpentelemetryEcto do
     end)
   end
 
+  defp maybe_add_db_operation(attributes, nil) do
+    attributes
+  end
+
+  defp maybe_add_db_operation(attributes, db_operation) do
+    Map.put(attributes, Trace.db_operation(), db_operation)
+  end
```

Comment:
In general, should I care about guarding against `nil,` or is that something that downstream would filter out? Or should I even bother sending a `nil` value in case it is nil?

I see `nil` values in my traces, so I want to confirm the general rule around the topic.